lib/cross_origen/ip_xact.rb

```diff
@@ -6,6 +6,39 @@ class IpXact < XMLDoc
 
     AddressBlock = Struct.new(:name, :base_address, :range, :width)
 
+    # Modified access type hash used in 1685-2009 (and 2014) schema
```

Comment:
Replicating this table here when it already exists in Origen core is a little bit of a code smell.
Could you consider an alternative implementation where the meta data you need here is added to `Origen::Registers::Bit::ACCESS_CODES` and then referenced from here?
See - https://github.com/Origen-SDK/origen/blob/master/lib/origen/registers/bit.rb#L8

Reply:
Fixed by updating Origen core in v0.38.0